tree/ntuple/v7/test/ntuple_view.cxx

```diff
@@ -54,6 +54,34 @@ TEST(RNTuple, View)
    EXPECT_EQ(3, n);
 }
 
+TEST(RNTuple, BulkView)
+{
+   FileRaii fileGuard("test_ntuple_bulk_view.root");
+
+   auto model = RNTupleModel::Create();
+   auto fieldPt = model->MakeField<float>("pt", 42.0);
+   auto eltsPerPage = 10'000;
+   {
+      RNTupleWriteOptions opt;
+      opt.SetNElementsPerPage(eltsPerPage);
+      auto ntuple = RNTupleWriter::Recreate(std::move(model), "myNTuple",
+         fileGuard.GetPath(), opt);
+      for (int i = 0; i < 100'000; i++) {
+         ntuple->Fill();
+      }
+   }
+   auto ntuple = RNTupleReader::Open("myNTuple", fileGuard.GetPath());
+   auto viewPt = ntuple->GetView<float>("pt");
+   NTupleSize_t nPageItems = 0;
+   const float *buf = viewPt.MapV(0, nPageItems);
```

Comment:
Let's add more cases to the test, in particular mapping at the end of the page (last and second last element).

Reply:
Thank you, added as part of the rebase e47da0a